source/api/commands/should.md

```diff
@@ -253,6 +253,21 @@ cy.get('#todos li').should(($lis) => {
 Read {% url 'Cypress should callback' https://glebbahmutov.com/blog/cypress-should-callback/ %} blog post to see more variations of the above example.
 {% endnote %}
 
+For clarity you can pass a string message as a second argument to any `expect` assertion.
+
+```javascript
+cy.get('.connectors-list > li').should(($lis) => {
+  expect($lis, '3 items').to.have.length(3)
+  expect($lis.eq(0), 'first item').to.contain('Walk the dog')
+  expect($lis.eq(1), 'second item').to.contain('Feed the cat')
+  expect($lis.eq(2), 'third item').to.contain('Write JavaScript')
+})
+```
+
+These string messages will be shown in the Command Log giving each assertion more context.
+
+![Expect assertions with messages](/img/api/should/expect-with-message.png)
+
```

Comment:
Oh, I learned something new. Is there a larger library documentation this can link to? To show where these description arguments come from?

Reply:
I will add link to https://www.chaijs.com/guide/styles/#expect